APMrover2/mode_acro.cpp

```diff
@@ -4,19 +4,19 @@
 void ModeAcro::update()
 {
     // get speed forward
-    float speed, desired_steering;
+    float speed, desired_steering, desired_lateral;
     if (!attitude_control.get_forward_speed(speed)) {
         float desired_throttle;
         // convert pilot stick input into desired steering and throttle
-        get_pilot_desired_steering_and_throttle(desired_steering, desired_throttle);
+        get_pilot_desired_steering_and_throttle(desired_steering, desired_throttle, desired_lateral);
```

Comment:
Should we pass the desired_lateral directly into the motors library in acro?  We can but some may think it's odd that in acro the pilot controls the speed forward but the lateral movement is manual.